lib/eex/lib/eex/engine.ex

```diff
@@ -98,21 +101,24 @@ defmodule EEx.Engine do
   @doc """
   Returns an empty string as initial buffer.
   """
-  def init(_opts) do
+  @spec init(Keyword.t) :: <<>>
+  def init(opts) when is_list(opts) do
```

Comment:
I agree than in this particular case the specs don't differ much, and I understand your point. 
I'm just trying to understand the reasoning behind this so I can avoid writing specs when not needed.
My only concern (and maybe this discussion should be moved to ExDoc, and maybe we already talked about this in on issue there) is that... by looking at the function itself, you have no way to know it's the default functions for the callback implementation.
So should we mention that in the docs?
Should we import the specs from callback and display them there, if no specs are available for the function.
thank you.